docs/internals/contributing/writing-code/coding-style.txt

```diff
@@ -215,8 +215,37 @@ Imports
 Template style
 ==============
 
-* In Django template code, put one (and only one) space between the curly
-  brackets and the tag contents.
+* In Django template code: 
+
+  The ``{% extends %}`` should be the first non-comment line
+
+  Do this:
+
+  .. code-block:: html+django
+
+      {% extends %}
+
+      Other code goes here
+
+  Or this:
+
+  .. code-block:: html+django
+
+      # This is a comment
```

Comment:
That's a _python_ comment, isn't it? It'd be `{# ... #}` (or, technically also `{% comment %} ... {% endcomment %}`) for the template language.

Reply:
Good catch :+1: Please use `{# .... #}`.